test/java/seedu/addressbook/commands/AddCommandTest.java

```diff
@@ -126,7 +126,7 @@ public void addCommand_emptyAddressBook_addressBookContainsPerson() {
         assertTrue(people.contains(p));
         assertEquals(1, people.immutableListView().size());
         assertFalse(result.getRelevantPersons().isPresent());
-        assertEquals(String.format(AddCommand.MESSAGE_SUCCESS, p), result.feedbackToUser);
+        assertEquals(String.format(AddCommand.MESSAGE_SUCCESS, p), result.getFeedbackToUser());
     }
 
```

Comment:
Nice to see Junit tests being updated (based on replacing public string with getter function). 
You could try to add tests for the added "Edit" functionality too.